app/cdap/components/WranglerGrid/SelectColumnPanel/CountWidget/__test__/SelectedColumnCountWidget.test.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+import CountWidget from 'components/WranglerGrid/SelectColumnPanel/CountWidget';
+import T from 'i18n-react';
+
+describe('It should test the SelectColumnsList Component', () => {
+  it('should render the SelectColumnsList Component with selectedColumnsCount>10', () => {
```

Comment:
if the behavior doesn't change with a greater number of selected columns count then why are there two tests?

Reply:
previously the logic was to add 0 when the count is less than 10. the the logic was changed and test file was updated accordingly. test is now updated.